css/main.css

```diff
@@ -22,7 +22,3 @@
 	
 }
 
-.headshot {
-    width: 250px;
-    height: 250px;
-}
```

Comment:
Were you able to confirm it wasn't a deployment issue? This seemed to work on my computer as well

Reply:
Not exactly sure how to check that again because we would have to push to production? I just manually resized the images this time.

Reply:
![image](https://user-images.githubusercontent.com/4440808/68412427-3076d500-015a-11ea-97ba-2dbab3d663ea.png)

Reply:
I just downloaded and checked it and something definitely is wrong.
![image](https://user-images.githubusercontent.com/4440808/68412591-7e8bd880-015a-11ea-9962-f63ed73e948a.png)

Reply:
Yeah so after downloading that build to my computer that build worked. Idk what happened. I l can just add the css back in ?

Reply:
Not sure, I want to understand what went wrong before we go back and forth.

Reply:
Can you try to isolate the issue first?

Reply:
I'd start by downloading what was deployed and compare those files.

Reply:
Yeah, so that build I just accidentally commented a few extra div tags out. Check pull request #14.

Reply:
The headshots class I added myself so I didn't have to resize the images, this new pull request should work and we don't need that css anymore